requirements.txt

```diff
@@ -1,7 +1,7 @@
 appdirs==1.4.0
 attrs==17.1.0
 click==6.6
-filesystems==0.14.0
+git+https://github.com/Julian/filesystems
```

Comment:
I cut a release here after merging your patches, I think 0.16.1 should also work. Lemme know if it doesn't.